scripts/travis.sh

```diff
@@ -14,7 +14,7 @@ if [ "$RUN_STATIC_CHECKS" ]; then
     fi
 fi
 
-make test
+NOMAD_SLOW_TEST=1 make test
```

Comment:
Define a make flag to optionally run these locally

Reply:
I am not sure why we need a make flag, eg I set TRAVIS=1 to pretend to be travis when running locally, and can similarly set NOMAD_SLOW_TEST=1 too.

Reply:
What does `TRAVIS=1` do? I know we intentionally lower the parallelism and raise timeouts when running on Travis which you do *not* want when running this slow test as it would only make it run more slowly.

I was trying to follow the example of `NOMAD_TEST_RKT` with this, although I guess their naming schemes differ.